client/client_session.cpp

```diff
@@ -74,6 +155,9 @@ void nibaclient::client_session::handle_cmd(const std::string &input) {
                 return create_and_go<nibashared::message_fight>(std::stoi(results[1]));
             } else if (results[0] == "learnmagic") {
                 return create_and_go<nibashared::message_learnmagic>(std::stoi(results[1]));
+            } else if (results[0] == "timeout") {
```

Comment:
In what case will handle_cmd("timeout") happen?

Reply:
https://github.com/td1tfx/niba2.0/pull/73/files#diff-1351f11973ce335c72238e096a15b0f6R49
test code here